modules/ROOT/partials/integrations/blazor-tech-ref.adoc

```diff
@@ -43,30 +43,32 @@ None of the configuration properties are required for the TinyMCE Blazor integra
 />
 ----
 
-=== CloudChannel
+=== `CloudChannel`
 
 Specifies the {cloudname} channel to use. For information on {productname} development channels, see: xref:editor-plugin-version.adoc[Specifying the {productname} editor version deployed from Cloud].
 
-*Default value:* `+'5'+`
+*Type:* `+String+`
+
+*Default value:* `'{productmajorversion}'`
 
-*Type:* String
+*Possible values:* `'{productmajorversion}-stable'`, `'{productmajorversion}-testing'`, `'{productmajorversion}-dev'`, `'{productminorversion}'`
 
 ==== Example using CloudChannel
 
-[source,cs]
+[source,cs,subs="attributes+"]
 ----
 <Editor
-  CloudChannel="5-dev"
+  CloudChannel="{productmajorversion}-dev"
 />
 ----
 
-=== Id
+=== `Id`
 
 Specified an Id for the editor. Used for retrieving the editor instance using the `+tinymce.get('ID')+` method.
 
-*Default value:* Automatically generated UUID
+*Type:* `+String+`
 
-*Type:* String
+*Default value:* Automatically generated UUID
```

Comment:
Just since I noticed we had a link elsewhere, do we care to add it here? 

> *Default value:* Automatically generated https://tools.ietf.org/html/rfc4122[UUID].

Reply:
Yeah I think we should to keep them consistent. I didn't check that part when I was fixing this.